csrc/quantization/compressed_tensors/int8_quant_kernels.cu

```diff
@@ -96,12 +96,14 @@ __global__ void static_scaled_int8_quant_kernel(
     scalar_t const* __restrict__ input, int8_t* __restrict__ out,
     scale_type const* scale_ptr, const int hidden_size) {
   int const tid = threadIdx.x;
-  int const token_idx = blockIdx.x;
+  int64_t const token_idx = blockIdx.x;
   scale_type const scale = *scale_ptr;
 
+  out += token_idx * hidden_size;
```

Comment:
one last nit: Could you add a comment noting that `token_idx * hidden_size` must be performed in 64-bit arithmetic?